### DISCUSSION TITLE
Add a NEI handler for the Smithing Table

### DISCUSSION THREAD

**Discussion by RaidhoSketch** [Roadhog360/Et-Futurum-Requiem#401] [Ideas]
Considering the CraftTweaker integration, it seems rither wise, in addition to generally providing some QoL in actually seeing the netherite tool recipes.

**Comment by Roadhog360** [comment]
This would be great too, unfortunately I don't know how to do that but will leave this open for anyone who is willing to contribute

**Comment by DarianLStephens** [comment]
I've already added some recipes with it, but would unfortunately need an external guide or quest book or something, because it's just not visible ingame.
If only NEI allowed you to create custom fake recipes for display purposes, I've wanted something like that for a long time. Place fake items in coordinates and have a backing texture, and it would be good enough for many uses.

**Comment by GregoriusT** [comment]
Yes Fake Recipes can definitely be added to NEI, technically speaking ALL recipe guis are fake, yes ALL of them, it just so happens that the Recipes they are based on tend to be real. NEI basically makes a copy of your Mods Recipe List to add it to its own Cache, ofcourse based on the Recipes you provide to it, see Smoker and Blast Furnace for example.

The problem only arises with Custom Recipe Handlers, but those could easily just fake an example Recipe that is provided with the Recipe Object itself. ^^
